### PR TITLE
More music

### PR DESCRIPTION
**Pull request by WTCWR68**
:cl: Todd
- add: Added a new lobby theme.

### CODE REVIEW

**Comment by moonheart08** [comment]
As this is from bandcamp please download it in ogg format directly instead of converting it.

**Comment by WTCWR68** [comment]
Fixed it so that it uses a directly downloaded File and the license is updated again.

File: Resources/Audio/Lobby/licenses.txt [hunk 1]
```diff
@@ -19,3 +19,5 @@ Marhaba.ogg (Marhaba) by Ian Alex Mac is licensed under a BY 4.0 Creative Common
 Comet-Haley.ogg (Comet Haley) by Stellardrone is licensed under a BY 3.0 Creative Commons License. It was converted from MP3 to OGG. The source can be found here: https://freemusicarchive.org/music/Stellardrone/Light_Years_1227/07_Comet_Halley
 
 Flip Flap.ogg (Flip Flap) by X-ceed is licensed by soundimage.org and is free for non-commercial use. It was converted from MP3 to OGG. The source is here http://aminet.net/package/mods/xceed/Flipflap
+
+Monument.ogg (Monument) by Six Umbrellas is licensed under a 4.0 Creative Commons License. It was downloaded as an OGG the source can be found here: https://sixumbrellas.bandcamp.com/album/the-psychedelic-and
```
Comment (Zumorica) [inline Resources/Audio/Lobby/licenses.txt:23]: The webpage linked there says this is licensed under the CC BY-SA 3.0

File: Resources/Audio/Lobby/licenses.txt [hunk 2]
```diff
@@ -14,10 +14,10 @@ vibe_ace.ogg (Vibe Ace) by Kevin MacLeod is licensed under CC-BY-3.0. It was con
...
+Marhaba.ogg (Marhaba) by Ian Alex Mac is licensed under a BY 3.0 Creative Commons License. It was converted from MP3 to OGG. The source can be found here: https://freemusicarchive.org/music/Ian_Alex_Mac/Cues/Marhaba
 
 Comet-Haley.ogg (Comet Haley) by Stellardrone is licensed under a BY 3.0 Creative Commons License. It was converted from MP3 to OGG. The source can be found here: https://freemusicarchive.org/music/Stellardrone/Light_Years_1227/07_Comet_Halley
 
 Flip Flap.ogg (Flip Flap) by X-ceed is licensed by soundimage.org and is free for non-commercial use. It was converted from MP3 to OGG. The source is here http://aminet.net/package/mods/xceed/Flipflap
 
-Monument.ogg (Monument) by Six Umbrellas is licensed under a 4.0 Creative Commons License. It was downloaded as an OGG the source can be found here: https://sixumbrellas.bandcamp.com/album/the-psychedelic-and
+Monument.ogg (Monument) by Six Umbrellas is licensed under a BY 3.0 Creative Commons License. It was downloaded as an OGG the source can be found here: https://sixumbrellas.bandcamp.com/album/the-psychedelic-and
```
Comment (Zumorica) [inline Resources/Audio/Lobby/licenses.txt:23]: it's licensed under the BY-SA license, not just BY

**Comment by WTCWR68** [comment]
license updated again.

File: Resources/Audio/Lobby/licenses.txt [hunk 3]
```diff
@@ -12,10 +12,12 @@ singuloose.ogg (Singuloose) by Janis Schiedková is licensed under a CC-BY-NC-SA
 
 vibe_ace.ogg (Vibe Ace) by Kevin MacLeod is licensed under CC-BY-3.0. It was converted from MP3 to OGG. The source can be found here: https://freemusicarchive.org/music/Kevin_MacLeod/Jazz_Sampler/Vibe_Ace
 
-Title3.ogg (Title3) By Cuboos is licensed under a BY-NC-SA 3.0 Creative Commons License. It is a remix of the song Tintin on the moon. It was converted from MP3 to OGG. The source can be found here: https://www.youtube.com/watch?v=YKVmXn-Gv0M
+Title3.ogg (Title3) By Cuboos is licensed under aCC-BY-NC-SA 3.0. It is a remix of the song Tintin on the moon. It was converted from MP3 to OGG. The source can be found here: https://www.youtube.com/watch?v=YKVmXn-Gv0M
```
Comment (Zumorica) [inline Resources/Audio/Lobby/licenses.txt:15]:
```suggestion
Title3.ogg (Title3) By Cuboos is licensed under a CC-BY-NC-SA 3.0. It is a remix of the song Tintin on the moon. It was converted from MP3 to OGG. The source can be found here: https://www.youtube.com/watch?v=YKVmXn-Gv0M
```

**Review by Zumorica — APPROVED** [review]

**Pull request merged**